tutorials/scripting/c_sharp/c_sharp_differences.rst

```diff
@@ -115,6 +126,8 @@ Example:
         private string _icon;
     }
 
+See also: :ref:`doc_c_sharp_exports`.
+
 Signal keyword
 --------------
```

Comment:
```suggestion
``signal`` keyword
------------------
```

Should we do it like this for consistency with the annotation formatting?

Reply:
Makes sense to me.